lib/rspec/core/option_parser.rb

```diff
@@ -68,7 +68,17 @@ def parser(options)
 
         parser.on('--bisect[=verbose]', 'Repeatedly runs the suite in order to isolate the failures to the ',
                   '  smallest reproducible case.') do |argument|
-          bisect_and_exit(argument)
+          options[:runner] = -> (opts, err, out) do
```

Comment:
We still need to support 1.8.7 so you'll have to use `Proc.new` or `lambda` instead of the 1.9+ stabby lambda syntax.